tests/test/test_add_item.py

```
@@ -0,0 +1,46 @@
+from tests.utilities.fixtures import login_with_volunteer
+from tests.utilities.fixtures import driver
+from tests.pages.add_item_page import AddItemPage
+import pytest
+from tests.pages.inventory_page import InventoryPage
+
+@pytest.fixture
+def inventory_page(driver):
```

Ruff (0.14.14)

Redefinition of unused `driver` from line 2: `driver` redefined here

(F811)